server/src/integration/java/org/opentosca/toscana/core/LogOverflowIT.java

```diff
@@ -0,0 +1,41 @@
+package org.opentosca.toscana.core;
+
+import java.io.File;
+
+import org.opentosca.toscana.core.transformation.logging.Log;
+import org.opentosca.toscana.core.transformation.logging.LogImpl;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+
+@SuppressWarnings("Duplicates")
+public class LogOverflowIT extends BaseIntegrationTest {
```

Comment:
Why is this not a unit test?

Reply:
Potential timeout issues. Took about 35 Seconds when running the first time.